plugins/CoreHome/vue/src/AjaxHelper/AjaxHelper.ts

```
@@ -68,8 +68,10 @@ type AnyFunction = (...params:any[]) => any; // eslint-disable-line
  * error callback to use by default
  */
 function defaultErrorCallback(deferred: XMLHttpRequest, status: string): void {
-  // do not display error message if request was aborted
-  if (status === 'abort' || !deferred || deferred.status === 0) {
+  // do not display error message if request was aborted or is unauthorized.
+  // We redirect to login page if status is unauthorized.
+  const noErrorMsgStatuses = [0, 401];
+  if (status === 'abort' || !deferred || noErrorMsgStatuses.includes(deferred.status)) {
```

I'm wondering if this could cause any possible regressions in plugin. The AjaxHelper might not only be used to load widgets or reports. Plugins cloud use it to load any external source. In case that fails with a 401, the error would now be hidden. Do we actually need to hide those errors? If we redirect, it shouldn't be shown for long anyway.

---

I've put back the original code. I agree that its not really needed that errors are hidden since we are redirecting
And its safer this way, than causing unnecessary side effects that could cause issues to others (Plugins)

Cheers